data/commodities.txt

```diff
@@ -112,43 +117,58 @@ trade
 		"linen sheets"
 		"mannequins"
 		"men's shoes"
+		"parkas"
 		"protective clothing"
 		"rayon"
 		"running shoes"
 		"sailcloth"
 		"socks"
 		"surgical scrubs"
+		"swimsuits"
 		"textiles"
 		"umbrellas"
 		"underwear"
 		"uniforms"
 		"winter coats"
 		"women's shoes"
+		"waterproof fiber"
 		"wool"
 		"work boots"
 	commodity "Metal" 190 590
 		"aluminized steel"
 		"aluminum"
+		"beryllium"
```

Comment:
You should probably remove any that are in-fact mineables. Otherwise you can end up with two of the same thing being listed as cargo. 
ie: 
Iron 10
Iron 10